src/data-card-advanced.tsx

```diff
@@ -0,0 +1,468 @@
+                                        {pretitle}
+                                    </Text2>
+                                )}
+                                <Text
+                                    aria-label="Texto"
+                                    color={vars.colors.textPrimary}
+                                    mobileSize={20}
+                                    mobileLineHeight="28px"
+                                    desktopSize={20}
+                                    desktopLineHeight="28px"
+                                    truncate={titleLinesMax}
+                                    weight={textPresets.cardTitle.weight}
+                                    as={titleAs}
+                                    hyphens="auto"
+                                >
+                                    {title}
+                                </Text>
+                                <Text2
+                                    aria-label="Texto"
+                                    color={vars.colors.textPrimary}
+                                    truncate={subtitleLinesMax}
+                                    as="p"
+                                    regular
+                                    hyphens="auto"
+                                >
+                                    {subtitle}
+                                </Text2>
+                            </Stack>
+                        </Stack>
+                    </header>
+                )}
+
+                {description && (
+                    <Text2
+                        truncate={descriptionLinesMax}
+                        as="p"
+                        regular
+                        color={vars.colors.textSecondary}
+                        hyphens="auto"
+                        aria-label="Texto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </div>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const flexDirection = hasAllItens ? 'column' : 'row';
+    const alignItems = hasAllItens ? 'start' : 'center';
+    const marginTop = hasAllItens ? '8px' : '16px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <NegativeBox>
+                <Divider />
```

Comment:
this divider is not working properly in desktop views
![image](https://github.com/Telefonica/mistica-web/assets/6722153/a39e5da2-ffef-474c-995c-ea6cc2b18a6c)


the divider has to fill the width